src/Pages/Projects/Projects.js

```diff
@@ -1,12 +1,26 @@
-import styles from './Projects.module.css';
+import styles from "./Projects.module.css";
+import projectsData from "./projectsData";
+import { useState } from "react";
+import Pagination from "./Pagination";
 
 const Projects = () => {
-    return (
-        <div className="Projects">
-            <h1>Projects</h1>
-        <p className={styles.projectStyle}>Here are a few projects we have worked on both individually and collaboratively</p>
-        </div>
-    );
-}
+    /*
+    I would probably have to implement pagination 
+    on this project section.
+    */
+   const [data, setData] = useState(projectsData)
+   return (
+    <div className={styles.Projects}>
+      <h2>Our Projects</h2>
+      <p className={styles.projectPara}>
+        Here are a few projects we have worked on both individually and
+        collaboratively.
+      </p>
+     
+         <Pagination data={data}
+         />
+    </div>
+  );
+};
```

Comment:
Hey You @Timothy-Achonu 
don't know if it was intentional, but you're not exporting some of your components like this one.